### ISSUE TITLE
Explicit license for Sim?

### ISSUE THREAD

**Issue by danbri** [tparisi/WebGLBook#9]
Would you consider posting an explicit license on the Sim js code? Perhaps same as Three's? ie. https://github.com/mrdoob/three.js/blob/master/LICENSE

**Comment by tparisi** [comment]
Good catch, I have been meaning to. Yes it will be an MIT-style like Three.js. Consider this your personal license grant under that license, if you need it, until I upload that.

**Comment by danbri** [comment]
Thanks!
